pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoader.java

```diff
@@ -849,16 +1054,24 @@ public UserDto createLegacyPepperUser(JdbiUser userDao, JdbiClient clientDao,
         long createdAtMillis = createdAtDate.toInstant(ZoneOffset.UTC).toEpochMilli();
         long updatedAtMillis = lastModifiedDate.toInstant(ZoneOffset.UTC).toEpochMilli();
 
-        String shortId = data.getAsJsonObject().get("ddp_participant_shortid").getAsString();
         String altpid = data.getAsJsonObject().get("datstat_altpid").getAsString();
+        JsonElement shortIdElem = data.getAsJsonObject().get("ddp_participant_shortid");
+        String shortId;
+        if (shortIdElem == null || shortIdElem.isJsonNull()) {
```

Comment:
Its safer to generate shortId in export file so that you can reconcile data if needed